src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/Sort.txt

```diff
@@ -0,0 +1,557 @@
+      { First: "Mary", Last: "Harris", Age: 48, Gender: "Female", Vaccinated: false, HasDog: false },
+      { First: "Titan", Last: Left("Hello", -1), Age: 792, Gender: "Unknown", Vaccinated: Blank(), HasDog: Blank() }),If(IsBlankOrError(Last),"",Last))
+Table({First:"Helio",Last:#Error(Kind=25),Age:263,Gender:"Unknown",Vaccinated:Blank(),HasDog:false},{First:"Titan",Last:#Error(Kind=25),Age:792,Gender:"Unknown",Vaccinated:Blank(),HasDog:false},{First:"John",Last:"Batali",Age:17,Gender:"Male",Vaccinated:false,HasDog:false},{First:"Mary",Last:"Harris",Age:48,Gender:"Female",Vaccinated:false,HasDog:false},{First:"Emily",Last:"Jones",Age:29,Gender:"Female",Vaccinated:true,HasDog:false},{First:"Bob",Last:"Smith",Age:2,Gender:"Male",Vaccinated:true,HasDog:true},{First:"Alice",Last:"Smith",Age:5,Gender:"Female",Vaccinated:true,HasDog:true})
+
+// SORT HIERARCHICAL DATA ON BOOL EXPRESSION - ASCENDING ORDER 
+
+>> Sort(Table(
+  { First: "Bob", Age: 2, Properties: { Vaccinated: true, HasDog: true }},
+  { First: "Alice", Age: 5, Properties: { Vaccinated: true, HasDog: true }},
+  { First: "John", Age: 17, Properties: { Vaccinated: false, HasDog: false }},
+  { First: "Emily", Age: 29, Properties: { Vaccinated: true, HasDog: Blank() }},
+  { First: "Helio", Age: 63, Properties: { Vaccinated: Blank(), HasDog: Blank() }},
+  { First: "Mary", Age: 48, Properties: Blank()},
+  { First: "Titan", Age: 79, Properties: Blank() }),Properties.HasDog)
+Table({Age:17,First:"John",Properties:{HasDog:false,Vaccinated:false}},{Age:2,First:"Bob",Properties:{HasDog:true,Vaccinated:true}},{Age:5,First:"Alice",Properties:{HasDog:true,Vaccinated:true}},{Age:29,First:"Emily",Properties:{HasDog:Blank(),Vaccinated:true}},{Age:63,First:"Helio",Properties:{HasDog:Blank(),Vaccinated:Blank()}},{Age:48,First:"Mary",Properties:Blank()},{Age:79,First:"Titan",Properties:Blank()})
+
+// SORT HIERARCHICAL DATA ON BOOL EXPRESSION - DESCENDING ORDER 
+
+>> Sort(Table(
+  { First: "Bob", Age: 2, Properties: { Vaccinated: true, HasDog: true }},
+  { First: "Alice", Age: 5, Properties: { Vaccinated: true, HasDog: true }},
+  { First: "John", Age: 17, Properties: { Vaccinated: false, HasDog: false }},
+  { First: "Emily", Age: 29, Properties: { Vaccinated: true, HasDog: Blank() }},
+  { First: "Helio", Age: 63, Properties: { Vaccinated: Blank(), HasDog: Blank() }},
+  { First: "Mary", Age: 48, Properties: Blank()},
+  { First: "Titan", Age: 79, Properties: Blank() }),Properties.HasDog, SortOrder.Descending)
+Table({Age:2,First:"Bob",Properties:{HasDog:true,Vaccinated:true}},{Age:5,First:"Alice",Properties:{HasDog:true,Vaccinated:true}},{Age:17,First:"John",Properties:{HasDog:false,Vaccinated:false}},{Age:29,First:"Emily",Properties:{HasDog:Blank(),Vaccinated:true}},{Age:63,First:"Helio",Properties:{HasDog:Blank(),Vaccinated:Blank()}},{Age:48,First:"Mary",Properties:Blank()},{Age:79,First:"Titan",Properties:Blank()})
+
+
+// SORT HIERARCHICAL DATA ON STRING EXPRESSION - ASCENDING ORDER 

```

Comment:
>TRING EXPRESSION [](http://example.com/codeflow?start=30&length=16)

The comment says that the sorting is done on a string expression, but it is actually done in a boolean expresion (same as two tests before that)

Reply:
Fixed the testcase to be consistent with the caption